media/css/firefox/releasenotes.less

```diff
@@ -1,40 +1,217 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 @import "../sandstone/lib.less";
+@import "//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css";
+@font-face {
+    font-family: 'Font Awesome';
+    src: url('/media/fonts/fontawesome-webfont.eot?v4.01#iefix') format('embedded-opentype'), url('/media/fonts/fontawesome-webfont.woff?v4.01') format('woff'), url('/media/fonts/fontawesome-webfont.ttf?v4.01') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+#outer-wrapper {
+    background: #fff;
+    width: 100%;
+    padding: 0px;
```

Comment:
A unit isn't needed for 0 length values, zero is just zero whether it's pixels or miles. This can just be <code>padding: 0;</code> and the same for everywhere else you're declaring a 0 length value.